src/Compilers/Core/Experimental/Microsoft.CodeAnalysis.Experimental.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Should we have `PublicAPI.Shipped.txt` and `PublicAPI.Unshipped.txt` for this project?

Reply:
I don't think so, this project contains only experimental APIs that we are free to change at any time.

Reply:
I could still imagine the files still being useful: that way you have confidence that you only exposed what you intended to expose. Otherwise we could accidentally expose more than you want, and then when we promote the API to a 'real' API we discover there was an implementation that was intended to be internal.